src/helpers/index.js

```diff
@@ -0,0 +1 @@
+export const getPrice = (value) => value.replace('R$', '').replace(',', '.');
```

Comment:
É melhor utilizar o Intl.NumberFormat, pq se o preço for maior do 1000 essa função não adiciona o ponto e a virgula.

Ex: R$ 1.000,00 

E para melhorar um pouco mais, você poderia chama a função com algum prefixo "format" ou algo do tipo.

Reply:
Blz

Reply:
Hm entendi. Vi aqui que nesse componente da para pegar o valor real do input.

Ex.: 
```javascript
const unmasked = this.cpfField.getRawValue()
// in the mask: 123.456.789-01
// unmasked: 12345678901
```

Reply:
Show